### PR TITLE
All-in-one node fixed, improved tooling and docs

### PR DESCRIPTION
**Pull request by djcp**

### CODE REVIEW

File: README.local-opsworks.md [hunk 1]
```diff
@@ -32,52 +32,68 @@ We do not currently support all-in-one nodes, but that should be coming soon.
 * Start your environment! This takes about 20 to 25 minutes on my beefy machine
   with SSD.
 
+        # Start an all-in-one node on http://10.10.10.50/
+        ./bin/all_in_one up
+        # Start a multi-node cluster on http://10.10.10.10/
```
Comment (karendolan) [inline README.local-opsworks.md:37]: Does this mean we can have both the all-in-one and the multi-node built, and suspend and resume between them?

Comment (djcp) [inline README.local-opsworks.md:37]: Yes it does!  

FWIW, that'd be possible without all the tooling I wrote in the `local_cluster` and `all_in_one` harnesses, but I think they should make things easier over all.

**Comment by karendolan** [comment]
Dan, I haven't given the changes for a spin, but they look good to me.

**Comment by lbjay** [comment]
:+1:

**Pull request merged**